xml/article_setup.xml

```diff
@@ -1194,51 +1220,40 @@ rmdir /dev/cpuset/$1
   </para>
 
   <para>
-   To set or retrieve the CPU affinity of a task a bitmask is used. It is
+   To set or retrieve the CPU affinity of a task, use a bitmask. This mask is
    represented by a hexadecimal number. If you count the bits of this bitmask,
-   the lowest bit represents the first logical CPU as they are found in
+   the lowest bit represents the first logical CPU as found in
    <filename>/proc/cpuinfo</filename>. For example:
   </para>
 
-  <variablelist>
-   <varlistentry>
-    <term><literal>0x00000001</literal></term>
-    <listitem>
-     <para>
-      is processor #0.
-     </para>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term><literal>0x00000002</literal></term>
-    <listitem>
-     <para>
-      is processor #1.
-     </para>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term><literal>0x00000003</literal></term>
-    <listitem>
-     <para>
-      is processor #0 and processor #1.
-     </para>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term><literal>0xFFFFFFFE</literal></term>
-    <listitem>
-     <para>
-      all but the first CPU.
-     </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     <literal>0x00000001</literal> means CPU zero.
```

Comment:
In this case, I'd be _much_ in favor of keeping the actual numbers rather than converting into numeral words. Other than than that, thanks! this list looks much better now.